--- /home/claude/card-forge/forge/forge-ai/src/main/java/forge/ai/simulation/GameCopier.java	2026-10-01 00:08:03.552943388 +0000
+++ patched/forge/ai/simulation/GameCopier.java	2026-10-01 00:20:48.482695111 +0000
@@ -293,7 +293,7 @@
     }
 
     private static PaperCard hidden_info_card = new PaperCard(CardRules.fromScript(Lists.newArrayList("Name:hidden", "Types:Artifact", "Oracle:")), "", CardRarity.Common);
-    private static final boolean PRUNE_HIDDEN_INFO = false;
+    private static boolean PRUNE_HIDDEN_INFO = false;
     private static final boolean USE_FROM_PAPER_CARD = true;
     private Card createCardCopy(Game newGame, Player newOwner, Card c, Player aiPlayer) {
         if (c.isToken() && !c.isImmutable()) {
